src/Components/DataHistoryExplorer/Internal/TimeSeriesViewer/TimeSeriesViewer.tsx

```diff
@@ -0,0 +1,308 @@
+        [t]
+    );
+    const updateXMinAndMax = useCallback(() => {
+        const nowInMillis = Date.now();
+        xMinDateInMillisRef.current =
+            nowInMillis - selectedQuickTimeSpanInMillis;
+        xMaxDateInMillisRef.current = nowInMillis;
+    }, [selectedQuickTimeSpanInMillis]);
+    const highChartSeriesData: Array<IHighChartSeriesData> = useMemo(
+        () =>
+            timeSeriesTwinList.length
+                ? transformADXTimeSeriesToHighChartsSeries(
+                      data,
+                      timeSeriesTwinList
+                  )
+                : [],
+        [data, timeSeriesTwinList]
+    );
+
+    // side effects
+    const prevQuery = usePrevious(query);
+    useEffect(() => {
+        if (query && query !== prevQuery) {
+            fetchTimeSeriesData();
+            updateXMinAndMax();
+        }
+    }, [query]);
+
+    // styles
+    const classNames = getClassNames(styles, {
+        theme: useTheme()
+    });
+
+    return (
+        <div className={classNames.root}>
+            <Pivot
+                overflowBehavior={'menu'}
+                styles={classNames.subComponentStyles.pivot}
+            >
+                <PivotItem
+                    headerText={t('dataHistoryExplorer.viewer.chart')}
+                    itemKey={ViewerPivot.Chart}
+                >
+                    {timeSeriesTwinList.length === 0 ? (
+                        <IllustrationMessage
+                            descriptionText={t(
+                                'dataHistoryExplorer.viewer.noSeriesDescription'
+                            )}
+                            type={'info'}
+                            width={'compact'}
+                            imageProps={{
+                                src: GenericErrorImg,
+                                height: 172
+                            }}
+                            styles={{ container: { flexGrow: 1 } }}
+                        />
+                    ) : (
+                        <>
+                            <Stack
+                                horizontal
+                                tokens={{ childrenGap: 8 }}
+                                className={classNames.commandWrapper}
+                            >
+                                <QuickTimesDropdown
+                                    defaultSelectedKey={getQuickTimeSpanKeyByValue(
+                                        selectedQuickTimeSpanInMillis
+                                    )}
+                                    onChange={(_env, option) =>
+                                        handleQuickTimesChange(option.data)
+                                    }
+                                    onRenderTitle={handleOnRenderQuickTimeTitle}
+                                    hasLabel={false}
+                                />
+                                <Dropdown
+                                    styles={{ root: { width: 88 } }}
```

Comment:
Please move to styles file

Reply:
I am going to change this to command bar in the subsequent PR, pending on this change now.